### PR TITLE
mobile recommendations

### PR DESCRIPTION
**Pull request by gringoireDM**

### CODE REVIEW

**Comment by rlmartin** [comment]
Relevant RFC: https://github.com/ModaOperandi/rfc/pull/18

**Comment by gringoireDM** [comment]
@ajpolt can you please have a look?

**Comment by ruxy** [comment]
Looks good!

**Comment by EdSancha** [comment]
@gringoireDM Are you considering `Combine` as part of SwiftUI? I feel it should be set as a framework of its own on Trial? WDYT?

File: recommendations/mobile.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+and supports a growing list of languages, the most recent addition being RxSwift. It also implements binding 
+to mobile and desktop platforms.
+
+### MERLin
+
+MERLin' is a reactive framework that aims to simplify the adoption of an event-driven architecture within an 
+iOS app. It emphasises the concept of modularity, promoting an easy to implement communication channel to deliver 
+events from producers to listeners.
+
+## Trial
```
Comment (ruxy) [inline recommendations/mobile.md]: @gringoireDM Does it make sense to add Bitrise under trial as well?

Comment (gringoireDM) [reply recommendations/mobile.md]: we still didn't try it, before putting it in trial I'd like the team to have a taste of it

Comment (rlmartin) [reply recommendations/mobile.md]: @gringoireDM to submit a PR proposing Bitrise into Assess for iOS builds only.

**Pull request merged**